docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,480 @@
+
+You will need to ensure that the load balancer routes to **just `master0` on port 6443**. This is because kubeadm will perform health checks using the load balancer IP. Since `master0` is set up individually first, the other masters will not have running apiservers, which will result in kubeadm hanging indefinitely.
+
+If possible, use a smart load balancing algorithm like "least connections", and use health checks so unhealthy nodes can be removed from circulation. Most providers will provide these features.
+
+## Acquire etcd certs
+
+Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**). If you are hosting etcd on the masters (**Option 2**), you can skip this step since you've already generated the etcd certificates on the masters.
+
+1. Generate SSH keys for each of the master nodes by following the steps in the [create ssh access](#create-ssh-access) section. After doing this, each master will have an SSH key in `~/.ssh/id_rsa.pub` and an entry in `etcd0`'s `~/.ssh/authorized_keys` file.
+
+1. Next, run the following:
+
+    ```shell
+    mkdir -p /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca.pem /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem /etc/kubernetes/pki/etcd
+    ```
+
+## Run `kubeadm init` on `master0` {#kubeadm-init-master0}
+
+1. In order for kubeadm to run, you first need to write a configuration file:
+
+    ```shell
+    cat >config.yaml <<EOL
+    apiVersion: kubeadm.k8s.io/v1alpha1
+    kind: MasterConfiguration
+    api:
+      advertiseAddress: <load-balancer-ip>
+    etcd:
+      endpoints:
+      - https://<etcd0-ip-address>:2379
+      - https://<etcd1-ip-address>:2379
+      - https://<etcd2-ip-address>:2379
+      caFile: /etc/kubernetes/pki/etcd/ca.pem
+      certFile: /etc/kubernetes/pki/etcd/client.pem
+      keyFile: /etc/kubernetes/pki/etcd/client-key.pem
+    networking:
+      podSubnet: <podCIDR>
+    apiServerCertSANs:
+    - <load-balancer-ip>
+    EOL
+    ```
+
+    Ensure that the following placeholders are replaced:
+
+    - `<load-balancer-ip>` with the public IPv4 of your load balancer. 
+    - `<etcd0-ip>`, `<etcd1-ip>` and `<etcd2-ip>` with the IP addresses of your three etcd nodes
+    - `<podCIDR>` with your Pod CIDR. Please read the [CNI network section](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network) of the docs for more information. Some CNI providers do not require a value to be set.
+
+1. When this is done, run kubeadm like so:
+
+    ```shell
+    kubeadm init --config=config.yaml
+    ```
+
+## Run `kubeadm init` on `master1` and `master2`
+
+Before running kubeadm on the other masters, you need to first copy the K8s CA cert from `master0`. To do this, you have two options:
+
+#### Option 1: Copy with scp
```

Comment:
Late comment:
We should consider if it is safer to copy all certificates except the API server cert (instead of copying only CA)
This will avoid e.g. having different sa.key on different masters

CC @luxas

Reply:
I can confirm that *at least* `/etc/kubernetes/pki/sa.{key,pub}` were required for me to get it to work.